Dockerfile.QA

```diff
@@ -1,4 +1,4 @@
-# Copyright 2018-2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# Copyright 2018-2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Should copyright be reverted?

Reply:
I am not changing it anymore... so yeah?